Data/Vector.hs

```diff
@@ -1743,6 +1746,19 @@ scanr1' :: (a -> a -> a) -> Vector a -> Vector a
 {-# INLINE scanr1' #-}
 scanr1' = G.scanr1'
 
+-- Comparisons
+-- ------------------------
+
+-- | /O(n)/ Check if two vectors are equal using supplied equality
+-- predicate.
+eqBy :: (a -> a -> Bool) -> Vector a -> Vector a -> Bool
+{-# INLINE eqBy #-}
+eqBy = G.eqBy
+
+-- | /O(n)/ Check if two vectors are equal using supplied comparison function
```

Comment:
It's important to mention here and above what happens for vectors of different length.

Reply:
I finally updated haddocks for cmpBy